readme.md

```diff
@@ -477,11 +477,11 @@ Make array a setoid:
 ```js
 Array.prototype.equals = (arr) => {
```

Comment:
No parens in [functor](https://github.com/hemanth/functional-programming-jargon#functor) though, now is time to discuss the merits and demerits of parens ;) ?